e2etest/zt_newe2e_blobfs_test.go

```diff
@@ -0,0 +1,244 @@
+		Flags: CopyFlags{
+			CopySyncCommonFlags: CopySyncCommonFlags{
+				Recursive: pointerTo(true),
+			},
+		},
+	})
+
+	ValidateResource[ContainerResourceManager](svm, dstContainer, ResourceDefinitionContainer{
+		Objects: srcObjs,
+	}, true)
+}
+
+func (s *BlobFSTestSuite) Scenario_DownloadFile(svm *ScenarioVariationManager) {
+	fileName := "test_1k.txt"
+	body := NewRandomObjectContentContainer(svm, common.KiloByte)
+
+	dstObj := CreateResource[ContainerResourceManager](svm, GetRootResource(svm, common.ELocation.Local()), ResourceDefinitionContainer{}).
+		GetObject(svm, fileName, common.EEntityType.File())
+
+	acct := GetAccount(svm, PrimaryHNSAcct)
+	srcService := acct.GetService(svm, common.ELocation.BlobFS())
+	srcContainer := CreateResource[ContainerResourceManager](svm, srcService, ResourceDefinitionContainer{})
+	srcObj := srcContainer.GetObject(svm, fileName, common.EEntityType.File())
+	srcObj.Create(svm, body, ObjectProperties{})
+
+	RunAzCopy(svm, AzCopyCommand{
+		Verb:    AzCopyVerbCopy,
+		Targets: []ResourceManager{srcObj.(RemoteResourceManager).WithSpecificAuthType(ResolveVariation(svm, []ExplicitCredentialTypes{EExplicitCredentialType.SASToken(), EExplicitCredentialType.OAuth()}), svm, CreateAzCopyTargetOptions{}), dstObj},
+		Flags: CopyFlags{
+			CopySyncCommonFlags: CopySyncCommonFlags{
+				Recursive: pointerTo(true),
+			},
+		},
+	})
+
+	ValidateResource[ObjectResourceManager](svm, dstObj, ResourceDefinitionObject{
+		Body: body,
+	}, true)
+}
+
+func (s *BlobFSTestSuite) Scenario_DownloadFile64MB(svm *ScenarioVariationManager) {
+	fileName := "test_64mb.txt"
+	body := NewRandomObjectContentContainer(svm, common.KiloByte)
+
+	dstObj := CreateResource[ContainerResourceManager](svm, GetRootResource(svm, common.ELocation.Local()), ResourceDefinitionContainer{}).
+		GetObject(svm, fileName, common.EEntityType.File())
+
+	acct := GetAccount(svm, PrimaryHNSAcct)
+	srcService := acct.GetService(svm, common.ELocation.BlobFS())
+	srcContainer := CreateResource[ContainerResourceManager](svm, srcService, ResourceDefinitionContainer{})
+	srcObj := srcContainer.GetObject(svm, fileName, common.EEntityType.File())
+	srcObj.Create(svm, body, ObjectProperties{})
+
+	RunAzCopy(svm, AzCopyCommand{
+		Verb:    AzCopyVerbCopy,
+		Targets: []ResourceManager{srcObj.(RemoteResourceManager).WithSpecificAuthType(EExplicitCredentialType.SASToken(), svm, CreateAzCopyTargetOptions{}), dstObj},
+		Flags: CopyFlags{
+			CopySyncCommonFlags: CopySyncCommonFlags{
+				Recursive: pointerTo(true),
+			},
+		},
+	})
+
+	ValidateResource[ObjectResourceManager](svm, dstObj, ResourceDefinitionObject{
+		Body: body,
+	}, true)
+}
+
+func (s *BlobFSTestSuite) Scenario_Download100Files(svm *ScenarioVariationManager) {
```

Comment:
Up/Download could be varied over

Reply:
For up/download I think I will skip varying over mostly due to the account type needing to be manually specified for hns.